File: java/day-25.java
import java.io.*;
import java.util.*;

public class Solution {

  public static void main(String[] args) {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */

    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();

    while (n-- > 0) {
      int num = sc.nextInt();

      if (isPrime(num)) {
        System.out.println("Prime");
      } else {
        System.out.println("Not prime");
      }
    }
  }

  private static boolean isPrime(int num) {
    if (num == 1) {
      return false;
    }

    int sqrtRoot = (int) Math.sqrt(Double.valueOf(num));

    for (int i = 2; i < sqrtRoot + 1; i++) {
      if (num % i == 0) {
        return false;
      }
    }

    return true;
  }
}

File: python/day-25.py
import math


def is_prime(num):
    if num == 1:
        return False

    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            return False

    return True


n = int(input())

while n:
    if is_prime(int(input())):
        print("Prime")
    else:
        print("Not prime")

    n -= 1
